model_compression_toolkit/quantizers_infrastructure/keras/README.md

```diff
@@ -0,0 +1,81 @@
+## Introduction
+
+[`BaseKerasTrainableQuantizer`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/keras/base_Keras_quantizer.py) is an interface that enables easy quantizers development and training. 
+Using this base class makes it simple to implement new quantizers for training and inference for weights or activations.
+
+## Make your own Keras trainable quantizers
+Trainable quantizer can be Weights Quantizer or Activation Quantizer.
+In order to make your new quantizer you need to create your quantizer class, `MyTrainingQuantizer` and do as follows:
+   - `MyTrainingQuantizer` should inherit from [`BaseKerasTrainableQuantizer`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/keras/base_Keras_quantizer.py).
+   - `MyTrainingQuantizer` should have [`init`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/quantizers_infrastructure/common/base_trainable_quantizer.py) function that gets `quantization_config` which is [`NodeWeightsQuantizationConfig`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/core/common/quantization/node_quantization_config.py) if you choose to implement weights quantizer or [`NodeActivationQuantizationConfig`](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/core/common/quantization/node_quantization_config.py) if you choose activation quantizer. This initialization function also defines:
```

Comment:
The references to the weights and activations are the same. It may be better to add the line to the references, so they get to the exact class.

Reply:
I agree. How do I do that?

Reply:
https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/core/common/quantization/node_quantization_config.py#L63

This is a link to line 63 for activation config (notice the suffix).

Reply:
But keep in mind that it's very fragile. Tomorrow this maybe different line.

Reply:
You're right, but I can't think of anything better since it's not exposed in our API